### DISCUSSION TITLE
想问下DNS不在自己本地或者临近城市正常吗？

### DISCUSSION THREAD

**Discussion by wwushi** [kkkgo/PaoPaoDNS#69] [问答和帮助] [answered]
您好，尊敬的网易用户
您的IP地址信息: 18*.***.***.**4 湖南省长沙市移动
您的DNS地址信息: 120.232.20.54 广东省广州市移动
您的DNS设置可能存在问题，请联系您的ISP服务商

想问下DNS不在自己本地或者临近城市正常吗？

**Comment by kkkgo** [comment, marked as answer]
正常的，如果开了CNFALL功能，首次请求为了加快响应速度一般会回落到转发，第二次访问就是递归的结果。
你多刷新几次说不定就可以看到IP地址和DNS地址是一样的。
当然，网易这个测试页面是生成他自己的随机字符串域名，刷新不一定能看到。

**Reply by kkkgo** [reply]
验证递归是否生效，根据文档使用`nslookup -type=TXT whoami.ds.akahelp.net`即可。

**Reply by kkkgo** [reply]
或者你可以把以下内容添加到`force_cn_list.txt`，可以看到一致的测试结果。
```yaml
domain:nstool.netease.com
domain:nstool.zhuanzfx.com
domain:nstool.laiqukankan.com
domain:nstool.321fenx.com
domain:nstool.haowu.link
domain:nstool.yqkk.link
```
